site2/docs/getting-started-clients.md

```diff
@@ -6,13 +6,16 @@ sidebar_label: Overview
 
 Pulsar supports the following client libraries:
 
-- [Java client](client-libraries-java.md)
-- [Go client](client-libraries-go.md)
-- [Python client](client-libraries-python.md)
-- [C++ client](client-libraries-cpp.md)
-- [Node.js client](client-libraries-node.md)
-- [WebSocket client](client-libraries-websocket.md)
-- [C# client](client-libraries-dotnet.md)
+|Language|Documentation|Release note|Code repo
+|---|---|---|---
+Java |- [User doc](client-libraries-java.md) <br /><br />- [API doc](https://pulsar.apache.org/api/client/)|Java client is released along with Pulsar, so Java client release note is a part of [Pulsar release note](https://pulsar.apache.org/release-notes/).|[Java client code repo](https://github.com/apache/pulsar/tree/master/pulsar-client) is hosted in the Pulsar main repo on GitHub. 
+C++ | - [User doc](client-libraries-cpp.md) <br /><br />- [API doc](https://pulsar.apache.org/api/cpp/)|C++ client is released along with Pulsar, so C++ client release note is a part of [Pulsar release note](https://pulsar.apache.org/release-notes/).|[C++ client code repo](https://github.com/apache/pulsar/tree/master/pulsar-client-cpp) is hosted in the Pulsar main repo on GitHub. 
+Python | - [User doc](client-libraries-python.md) <br /><br />- [API doc](https://pulsar.apache.org/api/python/)|Python client is released along with Pulsar, so Python client release note is a part of [Pulsar release note](https://pulsar.apache.org/release-notes/).|[Python client code repo](https://github.com/apache/pulsar/tree/master/pulsar-client-cpp/python) is hosted in the Pulsar main repo on GitHub. 
+WebSocket| [User doc](client-libraries-websocket.md) | WebSocket client is released along with Pulsar, so WebSocket client release note is a part of [Pulsar release note](https://pulsar.apache.org/release-notes/).|[WebSocket client code repo](https://github.com/apache/pulsar/tree/master/pulsar-websocket) is hosted in the Pulsar main repo on GitHub.
+Go client|[User doc](client-libraries-go.md)|Go client is not released along with Pulsar. The release note is available at [here](docs.google.com/document/d/1xiptFfsOvD1kHqGv6FyeMTD41R7Dffbhcz_1kCT_Rk0/edit#). |[Go client code repo](https://github.com/apache/pulsar-client-go) is hosted outside of Pulsar main repo.
```

Comment:
This link is to some other google doc and is in edit mode.

1. Release notes should be in a repository.
2. Release notes should not be in an immediately editable file format.

Reply:
@dave2wave Thanks for your suggestion! I've updated, PTAL